openedx/core/djangoapps/content/course_views/tabs.py

```diff
@@ -0,0 +1,573 @@
+"""
```

Comment:
I think ``content`` is a slightly confusing name for a Django app. Why did you pick that? I'm then confused why there's a ``course_views`` child folder. Was that intended to be the Django app name?